designer/server/src/models/forms/library.js

```diff
@@ -45,34 +57,73 @@ export function overviewViewModel(metadata) {
     form: metadata,
     formManagement: {
       heading: {
-        text: 'Form management',
+        text: 'Manage form',
         size: 'medium',
         level: '3'
       },
-      buttons: [
-        {
-          text: 'Edit draft',
-          href: `${formPath}/editor`,
-          classes: 'govuk-button--secondary-quiet'
-        }
-      ]
+      buttons,
+      submitButtons
     },
-    previewUrl: config.previewUrl
+    previewUrl: config.previewUrl,
+    notification
+  }
+}
+
+/**
+ * @param {boolean} displayCreateLiveSuccess - whether to display form live success message
+ * @param {boolean} displayCreateDraftSuccess - whether to display draft created success message
+ */
+function getFormOverviewNotification(
+  displayCreateLiveSuccess,
+  displayCreateDraftSuccess
+) {
+  if (displayCreateLiveSuccess) {
+    return 'This form is now live'
+  }
+
+  if (displayCreateDraftSuccess) {
+    return 'New draft created'
   }
 }
 
+/**
+ * @param {FormMetadata} metadata
+ * @param {string} baseUrl
+ */
+function getFormManagementButtons(metadata, baseUrl) {
+  const buttons = []
+  const submitButtons = []
+
+  if (metadata.draft) {
+    buttons.push(
+      {
+        text: 'Edit draft',
+        href: `${baseUrl}/editor`,
+        classes: 'govuk-button--secondary-quiet'
+      },
+      {
+        text: 'Make draft live',
+        href: `${baseUrl}/make-draft-live`
+      }
+    )
+  } else if (metadata.live) {
+    submitButtons.push({
+      text: 'Create draft to edit',
+      href: `${baseUrl}/create-draft-from-live`
```

Comment:
Shall we combine `buttons` + `submitButtons` but just use `action` instead of `href` here?

```suggestion
      action: `${baseUrl}/create-draft-from-live`
```